src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
```

Comment:
Let's add explicit type for state.
Check [this cheatset](https://mate-academy.github.io/fe-program/js/extra/react-typescript) and pay attention to `class component` section